pattern-2/marathon-applications/identity-server.json

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "wso2is",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "wso2is:5.4.1",
+      "network": "BRIDGE",
+      "portMappings": [
+        { "protocol": "tcp",
+          "hostPort": 0,
+          "labels": {
+            "VIP_0": "/wso2is:9763"
+          },
+          "containerPort": 9763,
+          "servicePort": 9763
+        },
+        { "protocol": "tcp",
+          "hostPort": 0,
+          "labels": {
+            "VIP_0": "/wso2is:9443"
+          },
+          "containerPort": 9443,
+          "servicePort": 9443
+        }
+      ],
+      "forcePullImage": false
+    },
+    "volumes": [
+      {
+        "containerPath": "/home/wso2carbon/volumes/",
+        "hostPath": "/volumes/wso2/identity-server/",
+        "mode": "RW"
+      }
+    ]
+  },
+  "env": {
+    "CLUSTERING_ENABLED": "false"
```

Comment:
As we discussed offline I think it might be better to remove this env variable as it may confuse users thinking that this may enable/disable clustering but it's actually used for updating localMemberHost value at the container startup.

Reply:
@imesh +1 for the suggestion. Will make the suggested changes as we progress with this implementation.